fable-test-app/src/components/NextHoliday.tsx

```diff
@@ -23,6 +29,27 @@ const NextHoliday: React.FC<NextHolidayProps> = ({
     return Math.floor((holidayDate.getTime() - today.getTime()) / (1000 * 3600 * 24));
   };
 
+  // Get long version of date
+  const nextHolidayDate = (dateString: string) => {
```

Comment:
I see potential for this to be generalized, especially seeing the function call is `nextHolidayDate(holidayDate)` when it only returns formatting. Can we change it?
```suggestion
  const formatDate = (dateString: string) => {
```